### PR TITLE
feat: Create 2022-07-01-verso-europa-digitale-sovrana-autonoma.md

### PR DESCRIPTION
**Pull request by Francescopog**
## Description
@bfabio  grazie

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Create 2022-07-01-verso-europa-digitale-sovrana-autonoma.md
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c2c5ecc52eac581a061ebf0c6d78006533c68e3c
</p>

File: _posts/it/2022-07-01-verso-europa-digitale-sovrana-autonoma.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Verso un’Europa digitale più autonoma e sovrana"
+subtitle: "Pubblicata la prima relazione del gruppo di lavoro sui digital commons. Il Dipartimento per la trasformazione digitale partecipa con Developers Italia nel team italiano"
+date: 2022-06-29 14:00:00 +0200
```
Comment (bfabio) [inline _posts/it/2022-07-01-verso-europa-digitale-sovrana-autonoma.md:5]:
```suggestion
date: 2022-07-01 18:00:00 +0200
```

File: _posts/it/2022-07-01-verso-europa-digitale-sovrana-autonoma.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+La Presidenza francese ha costituito il Gruppo di lavoro nel febbraio 2022 nell’ambito della conferenza [“Building Europe's Digital Sovereignty”](https://presidence-francaise.consilium.europa.eu/en/news/the-building-europe-s-digital-sovereignty-conference/#:~:text=The%20Building%20Europe's%20Digital%20Sovereignty%20conference%2C%20taking%20place%20on%207,continue%20building%20Europe's%20digital%20sovereignty), con il fine di creare un incubatore europeo sui digital commons, promuovere opportunità di finanziamento dedicate e fornire assistenza agli Stati Membri sulle tematiche connesse. L’iniziativa ha il fine ultimo di contribuire efficacemente allo sviluppo e **mettere a frutto il potenziale dei *digital commons* e dell’*open source***. Il Dipartimento per la trasformazione digitale porta l'esperienza di Developers Italia nel team italiano che ha lavorato al rapporto, composto dal Dipartimento per la trasformazione digitale, Ministero dello Sviluppo Economico e Ministero degli affari esteri e della cooperazione internazionale.
```
Comment (bfabio) [inline _posts/it/2022-07-01-verso-europa-digitale-sovrana-autonoma.md:14]:
```suggestion
La Presidenza francese ha costituito il Gruppo di lavoro nel febbraio 2022 nell’ambito della conferenza [“Building Europe's Digital Sovereignty”](https://presidence-francaise.consilium.europa.eu/en/news/the-building-europe-s-digital-sovereignty-conference/), con il fine di creare un incubatore europeo sui digital commons, promuovere opportunità di finanziamento dedicate e fornire assistenza agli Stati Membri sulle tematiche connesse. L’iniziativa ha il fine ultimo di contribuire efficacemente allo sviluppo e **mettere a frutto il potenziale dei *digital commons* e dell’*open source***. Il Dipartimento per la trasformazione digitale porta l'esperienza di Developers Italia nel team italiano che ha lavorato al rapporto, composto dal Dipartimento per la trasformazione digitale, Ministero dello Sviluppo Economico e Ministero degli affari esteri e della cooperazione internazionale.
```

File: _posts/it/2022-07-01-verso-europa-digitale-sovrana-autonoma.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+### Scopri di più e partecipa
+
+- [Scarica la relazione](https://www.diplomatie.gouv.fr/IMG/pdf/report_of_the_european_working_team_on_digital_commons_digital_assembly_june_2022_wnetherlands_cle843dbf.pdf) (disponibile in PDF solo in lingua inglese).
+
+- Vai sul sito del [Governo francese](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/diplomatie-numerique/actualites-et-evenements/article/le-rapport-sur-les-communs-numeriques-un-levier-essentiel-pour-la-souverainete) (solo in lingua francese)
+
+- Partecipa alla discussione su [Forum Italia](https://forum.italia.it/t/benvenuti-nel-forum-di-developers-italia/8) o su [Slack](https://developersitalia.slack.com/ssb/redirect)
```
Comment (bfabio) [inline _posts/it/2022-07-01-verso-europa-digitale-sovrana-autonoma.md:26]:
```suggestion
- Partecipa alla discussione su [Forum Italia](https://forum.italia.it/) o su [Slack](https://slack.developers.italia.it)
```

**Review by bfabio — CHANGES_REQUESTED** [review]

**Review by bfabio — APPROVED** [review]

**Pull request merged**